packages/styles/components/_c.tooltip.scss

```diff
@@ -19,17 +19,17 @@
   &__content {
     @include set-font-scale("04", "m");
 
-    background-color: $color-grey-700;
+    background-color: $color-tooltip-background;
     border-radius: get-border-radius("m");
     box-sizing: border-box;
-    color: $color-grey-000;
+    color: $color-font-lightest;
```

Comment:
Replace -lightest to -invert

Reply:
@caqueste as seen together, this point will be fixed in a new PR on the `@next` version